automations/python/label_pr.py

```diff
@@ -171,12 +177,38 @@ def get_label_of_cat(cat: str, labels: list[Label]) -> Label | None:
     return next(iter(get_all_labels_of_cat(cat, labels)), None)
 
 
+def get_stack_labels_from_changes(changes: list[str]) -> list[str]:
+    """
+    Given a list of changes identified by the `.github/actions/get-changes.yml` action,
+    this function returns a list of labels that must be applied to a PR.
+
+    :param changes: the changes identfied by the `get-changes` action
+    :return: the list of 'stack' labels to apply to the PR
+    """
+
+    labels = []
+    if "mgmt" in changes:
+        labels.append("🧱 stack: mgmt")
+        # Since the workflow appears in other changelists, we will immediately return.
```

Comment:
Tricky and well noticed. Would you mind adding a clarification of "the CI/CD workflow" specifically?